.drone.yml

```
@@ -242,8 +242,9 @@ volumes:
 
 trigger:
   branch:
-    - "staging"
-    - "staging-next"
+    exclude:
+      - "production"
+      - "test"
```

Our automated Deploy To Levelbuilder (DTL) process [creates a PR every day](https://github.com/code-dot-org/code-dot-org/pulls?q=is%3Apr+base%3Alevelbuilder+is%3Aclosed). We should add it to the list too.

```suggestion
    exclude:
      - "production"
      - "test"
      - "levelbuilder"
```